### PR TITLE
Add rate limit Exception `WebAuth.cli_login()`

### PR DESCRIPTION
**Pull request by sammiee5311**
I had a same issue(related to #316) with other person. 

I added few lines of code that handle the ratelimit !

### CODE REVIEW

File: steam/webauth.py [hunk 1]
```diff
@@ -224,6 +224,8 @@ def login(self, password='', captcha='', email_code='', twofactor_code='', langu
                 raise EmailCodeRequired(resp['message'])
             elif resp.get('requires_twofactor', False):
                 raise TwoFactorCodeRequired(resp['message'])
+            elif 'short time period' in resp.get('message', ''):
+                raise TooManyTimesFailed(resp['message'])
```
Comment (rossengeorgiev) [inline steam/webauth.py:228]: Can we rename `TooManyTimesFailed` to `TooManyLoginFailures`?

File: steam/webauth.py [hunk 2]
```diff
@@ -224,6 +224,8 @@ def login(self, password='', captcha='', email_code='', twofactor_code='', langu
                 raise EmailCodeRequired(resp['message'])
             elif resp.get('requires_twofactor', False):
                 raise TwoFactorCodeRequired(resp['message'])
+            elif 'short time period' in resp.get('message', ''):
```
Comment (rossengeorgiev) [inline steam/webauth.py:227]: Best to search for `too many login failures` instead here

**Review by rossengeorgiev — CHANGES_REQUESTED** [review]

File: steam/webauth.py (hunk 2 above)
Comment (sammiee5311) [reply steam/webauth.py]: > Best to search for `too many login failures` instead here

okay !

File: steam/webauth.py (hunk 1 above)
Comment (sammiee5311) [reply steam/webauth.py]: yeah, that is better naming.

**Review by sammiee5311 — COMMENTED** [review]
Thanks for the review !

**Review by rossengeorgiev — APPROVED** [review]
LGTM

**Pull request merged**